Updated libnotify 0.8.0-1 breaks notify-send

`notify-send` breaks after update `libnotify-0.8.0-1` package on Archlinux.

```
tests/test_notification.py:1: in <module>
    from notify import Notification
notify/__init__.py:1: in <module>
    from .notification import Notification
notify/notification.py:22: in <module>
    modulo = import_module("." + platform, __package__)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
notify/linux/__init__.py:1: in <module>
    from .notify import LinuxNotification
notify/linux/notify.py:10: in <module>
    gi.require_version("Notify", "0.7")
.venv/lib/python3.10/site-packages/gi/__init__.py:129: in require_version
    raise ValueError('Namespace %s not available for version %s' %
E   ValueError: Namespace Notify not available for version 0.7
```

--- a/notify/__init__.py
+++ b/notify/__init__.py
@@ -1,6 +1,6 @@
 from .notification import Notification
 from .notification import notification
 
-__version__ = '0.0.20'
+__version__ = '1.0.0'
 
 __all__ = ["Notification", "notification"]

--- a/notify/linux/notify.py
+++ b/notify/linux/notify.py
@@ -3,10 +3,11 @@ This module contains the code for the notification using Gtk api.
 
 https://lazka.github.io/pgi-docs/#Notify-0.7/classes/Notification.html
 https://lazka.github.io/pgi-docs/Notify-0.7/classes/Notification.html#Notify.Notification.set_app_name
+https://gitlab.gnome.org/GNOME/libnotify
 """
 import gi
 
-gi.require_version("Notify", "0.7")
+gi.require_version("Notify", "0.8")
 
 from gi.repository import Notify
 from gi.repository import GdkPixbuf

--- a/notify/notification.py
+++ b/notify/notification.py
@@ -20,8 +20,10 @@ platform = sys.platform
 
 try:
     modulo = import_module("." + platform, __package__)
-except Exception:
+except ModuleNotFoundError as exc:
     raise RuntimeError("Unsupported operating system: {}".format(platform))
+except Exception:
+    raise
 else:
     caller = getattr(modulo, "{}Notification".format(platform.title()))()
 

--- a/notify/notification.py
+++ b/notify/notification.py
@@ -35,7 +35,7 @@ class Notification:
         message (str): The message body (optional).
         timeout (int): Notification length in milliseconds (optional).
         app_name (str): Caller app name. Defaults to 'notify-send'.
-        **kwargs: Additional arguments (optional).
+        kwargs: Additional arguments (optional).
     """
 
     def __init__(self, summary, message="", timeout=2000, **kwargs):

--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,7 @@ def readme():
 
 setup(
     name="notify-send",
-    version="0.0.20",
+    version="1.0.0",
     description="notify-send notify.",
     long_description=readme(),
     long_description_content_type="text/markdown",

